docs/docker-stack/entrypoint.rst

```diff
@@ -131,7 +131,7 @@ The entrypoint can also create webserver user automatically when you enter it. y
 ``_AIRFLOW_WWW_USER_CREATE`` to a non-empty value in order to do that. This is not intended for
 production, it is only useful if you would like to run a quick test with the production image.
 You need to pass at least password to create such user via ``_AIRFLOW_WWW_USER_PASSWORD_CMD`` or
-``_AIRFLOW_WWW_USER_PASSWORD_CMD`` similarly like for other ``*_CMD`` variables, the content of
+``_AIRFLOW_WWW_USER_PASSWORD`` similarly like for other ``*_CMD`` variables, the content of
```

Comment:
Thanks @raphaelauv .

I agree this is a typo. But by reading through the sentence, I think the order of ``_AIRFLOW_WWW_USER_PASSWORD_CMD`` and ``_AIRFLOW_WWW_USER_PASSWORD`` should be swapped in your change.

Reply:
done , thank you

Reply:
Merci :)